### PR TITLE
Update encoder-decoder notebooks

### PR DESCRIPTION
**Pull request by takumiohym**
Update encoder-decoder notebooks:
- Removed NLTK dependency and updated the broken BLEU score logic with Hugging Face evaluate lib (also 100x speed up)
- Removed the old `.h5` format

### CODE REVIEW

File: notebooks/text_models/labs/rnn_encoder_decoder.ipynb [hunk 1]
```diff
@@ -20,15 +20,6 @@
     "At last, we'll benchmark our results using the industry standard BLEU score."
```
Comment (sanjanalreddy) [inline notebooks/text_models/labs/rnn_encoder_decoder.ipynb:20]: <p>fix typo: "evaluagte" -&gt; evaluate</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/522/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='70'/>

Comment (sanjanalreddy) [inline notebooks/text_models/labs/rnn_encoder_decoder.ipynb:20]: <p>Typo: unitl -&gt; until</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/522/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='72'/>

**Review by sanjanalreddy — APPROVED** [review]
LGTM, please fix typos

File: notebooks/text_models/labs/rnn_encoder_decoder.ipynb (hunk 1 above)
Comment (takumiohym) [reply notebooks/text_models/labs/rnn_encoder_decoder.ipynb:20]: <p>Fixed. Thanks!</p>

Comment (takumiohym) [reply notebooks/text_models/labs/rnn_encoder_decoder.ipynb:20]: <p>Fixed. Thanks!</p>

**Pull request merged**